app/assets/stylesheets/application.css

```diff
@@ -1,2 +1 @@
-@import url('bundle.min.css');
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800');
```

Comment:
Is the Open Sans font still downloaded from somewhere else? (since it's not usually a preinstalled font)

Reply:
yes it is! in app/assets/src/styles/themes/_typography.scss